### PR TITLE
Feat/funding

### PR DESCRIPTION
**Pull request by ElTimuro**

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

File: apps/marketing/src/app/(marketing)/open/page.tsx [hunk 1]
```diff
@@ -70,9 +70,12 @@ export default async function OpenPage() {
       <div className="flex flex-col items-center justify-center">
         <h1 className="text-3xl font-bold lg:text-5xl">Open Startup</h1>
 
-        <p className="text-muted-foreground mt-4 max-w-[55ch] text-center text-lg leading-normal">
+        <p className="text-muted-foreground mt-4 max-w-[60ch] text-center text-lg leading-normal">
           All our metrics, finances, and learnings are public. We believe in transparency and want
-          to share our journey with you.
+          to share our journey with you. You can read more about why here:{' '}
+          <a className="font-bold" href="https://documenso.com//blog/preseed" target="_blank">
```
Comment (dephraiim) [inline apps/marketing/src/app/(marketing)/open/page.tsx:76]: One black slash instead

`https://documenso.com/blog/preseed`

Comment (ElTimuro) [reply apps/marketing/src/app/(marketing)/open/page.tsx]: good catch

**Comment by ElTimuro** [comment]
merging and migrating to website fork

**Pull request merged**